webviews/components/IssueCard.svelte

```diff
@@ -0,0 +1,17 @@
+<script>
+  const deleteCard = () => {
+    //add mutation for deleting card
```

Comment:
I see you've made this easy for me by adding these. Just a quick question, how can one access that card
s properties inside this anonymous function?

Reply:
We can pass it as a prop from the parent svelte component. I can do it if you can tell me what card's properties you need.

Reply:
I need the whole card, as well as a payload (from user)

For DeleteCard, we just need the card. For things like EditCard, we need as I stated above.

Reply:
Take a look at the `handleCardMutations(card, request, payload)`